File: src/main/java/com/learnershi/rclasssocket/entity/Draw.java
package com.learnershi.rclasssocket.entity;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.Accessors;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

/**
 * Hands-Up
 *
 * @author GyeongSik
 */
@Accessors(chain = true)
@NoArgsConstructor
@Getter
@Setter
@Document(collection = "draw")
public class Draw extends ClassRoomEntity {
    @Id
    private String id;
    private String color;
    private String opacity;
    private String width;
    private List path;
    private String writerWidth;
}

File: src/main/java/com/learnershi/rclasssocket/entity/Draw.kt
package com.learnershi.rclasssocket.entity

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

/**
 * Hands-Up
 *
 */
@Document(collection = "draw")
class Draw : ClassRoomEntity() {
    @Id
    private val id: String? = null
    private val color: String? = null
    private val opacity: String? = null
    private val width: String? = null
    private val path: List<*>? = null
    private val writerWidth: String? = null
}